app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -300,13 +315,43 @@ class SectionDateHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBi
     }
 }
 
-class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBinding) :
+class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxConflictSectionHeaderBinding) :
     BaseTransactionViewHolder<TransactionView.SectionConflictHeader>(viewBinding) {
 
     override fun bind(sectionDateHeader: TransactionView.SectionConflictHeader, payloads: List<Any>) {
+        val resources = viewBinding.root.context.resources
+
         with(viewBinding) {
-            sectionTitle.text = sectionDateHeader.nonce.toString()
+            nonce.text = sectionDateHeader.nonce.toString()
+            sectionTitle.text = resources.getString(R.string.tx_list_conflict_header_explainer)
+            sectionTitle.appendLink(
+                resources.getString(R.string.tx_list_conflict_header_link),
+                resources.getString(R.string.tx_list_conflict_header_learn_more)
+            )
+        }
+    }
+
+    private fun TextView.appendLink(url: String, urlText: String) {
```

Comment:
This method looks like it could be separated into 2: `prepareLinkSpannable` and `appendLink`. 

I think it would be nice to extract these extension functions into a `StringUtils` file. The same would for `SpannableStringBuild.appendTextWithSpans`

Reply:
See new version in: 2a1cf57848f392a2611397c6554133af37f44517